src/constants/options/status.ts

```diff
@@ -89,3 +89,15 @@ export const getChangeRequestStatusOptions = (t: typeof useT | Function = (t: st
       title: t("No update")
     }
   ] as Option[];
+
+export const getNothingReportOptions = (t: typeof useT | Function = (t: string) => t) =>
+  [
+    {
+      value: "reportInfo",
+      title: t("Reported Info")
+    },
+    {
+      value: "nothingToReport",
```

Comment:
Oh, this helps explain some of what's going on in the v3 controller, but why use these arbitrary strings instead of a simple boolean?